src/fiatconnect/KycLanding.tsx

```diff
@@ -12,7 +14,20 @@ export interface Props {
 }
 
 function KycLanding(_props: Props) {
-  return <SafeAreaView style={styles.container} />
+  const sendKYCSchema = async () => {
+    if (!_props.quote.quoteResponseKycSchema) {
```

Comment:
The `_` prefix is typically only used when the variable is unused in the code. Now that you are using it you can remove it from the `_props` variable name

Reply:
Instead of digging into the properties of `quote` directly you should use the helper functions on the quote like `getKycSchema`. Especially if they already exist